src/registry.jl

```diff
@@ -188,7 +188,23 @@ function detectavi(io)
     submagic == b"AVI "
 end
 add_format(format"AVI", detectavi, ".avi", [idImageMagick], [idVideoIO])
-add_format(format"MP4", UInt8[0x00,0x00,0x00,0x18,0x66,0x74,0x79,0x70], ".mp4", [idVideoIO])
+
+""" detectisom(io)
+
+Detect ISO/IEC 14496-12 ISO/IEC base media format files. These files start with
+a 32-bit big-endian length, and then the string 'ftyp' which is followed by
+details of the container and codec. Finding 'ftyp' is enough to know to dispatch
+to VideoIO.
```

Comment:
> Finding 'ftyp' is enough to know to dispatch to VIdeoIO

I'm not an expert on video formats. Are there any references for this that can help me/others to clarify this?

Reply:
I'm not an expert either, I used these three references to come to this conclusion:
Wikipedia has a readable introduction: https://en.wikipedia.org/wiki/ISO/IEC_base_media_file_format#File_type_box

There is a publicly available version of the spec here: https://standards.iso.org/ittf/PubliclyAvailableStandards/c068960_ISO_IEC_14496-12_2015.zip

As a check, I compared to the C++ code here: https://github.com/DigiDNA/ISOBMFF